platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -1658,6 +2286,10 @@ namespace platoon_strategic_ihp
             request.strategy_params = "";
             request.urgency = 50;
             request.location = pose_to_ecef(pose_msg_);
+            
+            // UCLA added for cut-in, not used for other cases
+            request.join_index = -1; // Note: not used by same-lane functions.
```

Comment:
Seems to indicate that this will always be a front join.  How does that work when plan_type implies host will be a follower (in fact, this method name implies that this is only for followers, not a future leader)?